NewRelicBrowser.php

```diff
@@ -1,392 +1,456 @@
+							$relic_account_email = ( isset( $_POST['relic-account-email'] ) ) ? sanitize_email( wp_unslash( $_POST['relic-account-email'] ) ) : '';
+							$relic_first_name    = ( isset( $_POST['relic-first-name'] ) ) ? sanitize_text_field( wp_unslash( $_POST['relic-first-name'] ) ) : '';
+							$relic_last_name     = ( isset( $_POST['relic-last-name'] ) ) ? sanitize_text_field( wp_unslash( $_POST['relic-last-name'] ) ) : '';
+
+							$data = array(
+								'account' => array(
+									'name'             => $relic_account_name,
+									'allow_api_access' => true,
+									'testing'          => false,
+									'users'            => array(
+										array(
+											'email'      => $relic_account_email,
+											'password'   => $relic_password,
+											'first_name' => $relic_first_name,
+											'last_name'  => $relic_last_name,
+											'role'       => 'admin',
+											'owner'      => 'true',
+										),
 									),
+									'subscriptions'    => array(),
 								),
-								'subscriptions' => array(),
-							)
-						);
-
-						/* for this api data is to be pass in json */
-						$dataString = json_encode( $data );
-
-						$create_browser_response = wp_remote_post(
-							RTP_NEW_RELIC_API_URL, array(
-							'timeout' => 100,
-							'method' => 'POST',
-							'headers' => array(
-								'x-api-key' => RTP_NEW_RELIC_API_KEY,
-								'Content-Type' => 'application/json',
-							),
-							'body' => $dataString,
-								) );
+							);
+
+							/* for this api data is to be pass in json */
+							$datastring = wp_json_encode( $data );
+
+							$create_browser_response = wp_remote_post(
+								RTP_NEW_RELIC_API_URL,
+								array(
+									'timeout' => 100,
+									'method'  => 'POST',
+									'headers' => array(
+										'x-api-key'    => RTP_NEW_RELIC_API_KEY,
+										'Content-Type' => 'application/json',
+									),
+									'body'    => $datastring,
+								)
+							);
+
 							$create_browser_response_code = wp_remote_retrieve_response_code( $create_browser_response );
+
 							$json_data = json_decode( $create_browser_response['body'] );
-							if ( 201 != $create_browser_response_code ) {
+							if ( 201 !== (int) $create_browser_response_code ) {
 								add_settings_error( 'relic_options', 'relic_options_error', $json_data->error, 'error' );
-							} else if ( isset( $json_data->api_key ) ) {
+							} elseif ( isset( $json_data->api_key ) ) {
 								/* store the received data */
 								$main_array = array(
 									'relic_account_name' => $json_data->name,
-									'relic_api_key' => $json_data->api_key,
-									'relic_id' => $json_data->id,
-									'relic_password' => $relic_password,
+									'relic_api_key'      => $json_data->api_key,
+									'relic_id'           => $json_data->id,
+									'relic_password'     => $relic_password,
 								);
 								add_option( $option_name, $main_array );
```

Comment:
Use `update_option()`

Reply:
i solved this issue. please review changes.